homeassistant/components/opengarage/cover.py

```diff
@@ -55,7 +56,7 @@
 )
 
 
-def setup_platform(hass, config, add_entities, discovery_info=None):
+async def async_setup_platform(hass, config, add_entities, discovery_info=None):
```

Comment:
Please break out this into a separate PR so we can include that in a patch release, as it fixes a bug.

Reply:
Done: https://github.com/home-assistant/core/pull/36075

Reply:
Great! When that is merged we can rebase here.